pages/devops-resources.md

```diff
@@ -110,4 +111,5 @@
 ## GitHub Repositories to learn DevOps
 |Description | Preview   | Link to get started with it   |
 | :------------: | :------------: | :------------: |
-|  1. 90 Days of DevOps - Roadmap  | <center> [![open source](https://ik.imagekit.io/1cw2zpbjy/OSWH/Getting_started_with_DevOps_-_Banner.png?ik-sdk-version=javascript-1.4.3&updatedAt=1667725617154 "DevOps")](https://github.com/MichaelCade/90DaysOfDevOps "90 Days of DevOps - Roadmap") </center> |   [Here you go](https://github.com/MichaelCade/90DaysOfDevOps)
\ No newline at end of file
+|  1. 90 Days of DevOps - Roadmap  | <center> [![open source](https://ik.imagekit.io/1cw2zpbjy/OSWH/Getting_started_with_DevOps_-_Banner.png?ik-sdk-version=javascript-1.4.3&updatedAt=1667725617154 "DevOps")](https://github.com/MichaelCade/90DaysOfDevOps "90 Days of DevOps - Roadmap") </center> |   [Here you go](https://github.com/MichaelCade/90DaysOfDevOps)
+|  2. DevOps Learning Hub: Tools and Tech  | <center> [![open source](https://ik.imagekit.io/hfclyehqd/OS_Bag/DevOps_-_Pradumnasaraf.jpeg?updatedAt=1684313490774 "DevOps")](https://github.com/Pradumnasaraf/DevOps "DevOps Learning Hub: Tools and Tech") </center> |   [Here you go](https://github.com/Pradumnasaraf/DevOps)
```

Comment:
Edit the description and include pradumna sarafs name in it @mhdamaan79

Reply:
**Description:**
DevOps Learning Hub: Tools and Tech - Pradumna Saraf

Just like this?

Reply:
YEs @mhdamaan79

Reply:
Updated description with Pradumna Saraf's name included. Please review the changes @hasnainmakada-99